### PR TITLE
Update README.md for mainnet

### PR DESCRIPTION
**Pull request by mat7ias**
This includes changes: 
* adds link to website (along with Golem image) and badges
* replace some descriptions from the old website with more recent descriptions. 
* adds a paragraph about developers (under providers and requestors)
* mentions JS api
* makes the README more mainnet relevant (links to blog and v0.6.1 release)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,48 @@
...
 
 * **Requestor**
 Has a need to use IT resources such as computation hardware. Those resources are purchased in the decentralized market. The actual usage of the resources is backed by Golem's decentralized infrastructure.
 
 * **Provider**
 Has IT resources available that can be shared with other actors in the network. Those resources are sold in the decentralized market.
 
+* **Developer**
+Builds applications to run for requestors on the network. Golem's potential goes much beyond a singular application. In Golem there is not a single monolithic use-case - Golem allows for a self sustainable ecosystem. A myriad of different types of users requesting digital resources, providing them and building applications on top of the platform. See [Awesome Golem](https://github.com/golemfactory/awesome-golem/blob/main/README.md#%EF%B8%8F-apps) for just a taste of the various types of applications that can be built and run on Golem!
```
Comment (tworec) [inline README.md:41]: I do not get sense of the below sentence within this context. Can you rephrase? Or enlighten me? :)

> A myriad of different types of users requesting digital resources, providing them and building applications on top of the platform

File: README.md [hunk 2]
```diff
@@ -42,8 +70,9 @@ We call our runtime **ExeUnit**. As for now we support
 
 Other ExeUnit types are to come (see below).
 
-## MVP Requirements
+## MVP
 
+With the MVP out, in the form of the Beta I ([Grace Hopper](https://github.com/golemfactory/yagna/releases/tag/v0.6.1)) release. The release includes:
```
Comment (tworec) [inline README.md:75]:
```suggestion
With the MVP out, in the form of the ([Beta I Grace Hopper](https://github.com/golemfactory/yagna/releases/tag/v0.6.1)) release. The release includes:
```

File: README.md (hunk 1 above)
Comment (mat7ias) [reply README.md]: Good point! I've made it more concise.

**Review by tworec — APPROVED** [review]
